src/OrchardCore.Modules/OrchardCore.Tenants/ViewModels/CreateApiViewModel.cs

```diff
@@ -2,6 +2,6 @@ namespace OrchardCore.Tenants.ViewModels
 {
     public class CreateApiViewModel : TenantViewModel
     {
-
+        public string Category { get; set; }
```

Comment:
Maybe better to put it in the base `TenantViewModel` if needed in both derived `CreateApiViewModel ` and `EditTenantViewModel`.